src/tracks/playabletrack/notetrack/ui/NoteTrackAffordanceControls.cpp

```diff
@@ -82,6 +84,19 @@ std::vector<UIHandlePtr> NoteTrackAffordanceControls::HitTest(const TrackPanelMo
     {
         results.push_back(NoteTrackAffordanceHandle::HitAnywhere(mAffordanceHandle, track));
     }
+    else
+    {
+        const auto& settings = ProjectSettings::Get(*pProject);
+        const auto currentTool = settings.GetTool();
+        if (currentTool == ToolCodes::multiTool || currentTool == ToolCodes::selectTool)
+        {
+            results.push_back(
+                SelectHandle::HitTest(
+                    mSelectHandle, state, pProject, std::static_pointer_cast<TrackView>(track->GetTrackView())
+                )
+            );
+        }
```

Comment:
If instead you push this handle uncondtionally, then it will be possible to start a selection drag even if the mouse is inside a clip affordance -- if you hit Escape before the click and drag.

A minor advantage that few users may discover, but why not do it, for small extra effort.

Reply:
Thanks! Will fix that. Also currently it is not possible to make selection in affrodances area while one of the clips is beeing edited, will fix that too

Reply:
I see the new round of changes.  How is this problem fixed?